src/main/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperation.java

```diff
@@ -115,31 +136,30 @@ public StartEventTrigger triggerStartForDraft(final String uid,
                                                   final String draftReference,
                                                   final String eventTriggerId,
                                                   final Boolean ignoreWarning) {
-        return buildStartEventTrigger(uid,
-                                      jurisdictionId,
-                                      caseTypeId,
+        final CaseType caseType = getCaseType(caseTypeId);
+
+        return buildStartEventTrigger(() -> uid,
+                                      caseType,
                                       eventTriggerId,
                                       ignoreWarning,
-            () -> getDraftDetails(draftReference));
+                                      () -> getDraftDetails(draftReference));
     }
 
-    private StartEventTrigger buildStartEventTrigger(final String uid,
-                                                     final String jurisdictionId,
-                                                     final String caseTypeId,
+    private StartEventTrigger buildStartEventTrigger(final Supplier<String> uidSupplier,
+                                                     final CaseType caseType,
                                                      final String eventTriggerId,
                                                      final Boolean ignoreWarning,
                                                      final Supplier<CaseDetails> caseDetailsSupplier) {
-        final CaseType caseType = getCaseType(caseTypeId);
-        final CaseEvent eventTrigger = getEventTrigger(caseTypeId, eventTriggerId, caseType);
+        final CaseEvent eventTrigger = getEventTrigger(caseType.getId(), eventTriggerId, caseType);
 
-        validateJurisdiction(jurisdictionId, caseTypeId, caseType);
+        validateJurisdiction(caseType.getJurisdictionId(), caseType.getId(), caseType);
```

Comment:
I don't think this is required anymore. If the jurisdiction is always taken from the case type, then validating it is not required.

Reply:
Good point